addons/a11y/src/components/ColorBlindness.tsx

```diff
@@ -86,45 +79,39 @@ const getColorList = (active: string | null, set: (i: string | null) => void): L
   })),
 ];
 
-export class ColorBlindness extends Component<ColorBlindnessProps, ColorBlindnessState> {
-  state: ColorBlindnessState = {
-    active: null,
-  };
+export function ColorBlindness() {
```

Comment:
Is typing this redundant?

```
export const ColorBlindness: React.FC<{}>) = () => { ... }
```

Reply:
I guess it depends if you want to explicitly type everything, I guess personal preference?

Reply:
Haha thanks! Wasn't requesting--was really a question about TypeScript best practices. I'm still a newbie.

Reply:
@shilman sorry! it's still personal preference, Typescript should be able to figure it out, but it doesn't hurt.  If in the tsconfig, the `noImplicitReturns` rule is enforced, then you need to list the return values